packages/components/pages/f-offers/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-offers",
   "description": "Fozzie Offers - displays offers to the customers",
-  "version": "2.1.0-beta.1",
+  "version": "1.12.0-beta.4",
```

Comment:
Make sure to add `tag: "beta"`

Reply:
How come? Is that how lerna finds the tag?